pkgs/development/python-modules/textx/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, python3
+, fetchFromGitHub
+, mkdocs
+, twine
+, arpeggio
+, click
+, future
+, coverage
+, coveralls
+, flake8
+, html5lib
+, jinja2
+, pytest
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "textx";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-uZlO82dKtWQQR5+Q7dWk3+ZoUzAjDJ8qzC4UMLCtnBk=";
+  };
+
+  nativeBuildInputs = [
+    mkdocs
+    twine
+  ];
+
+  propagatedBuildInputs = [
+    arpeggio
+    click
+    future
+  ];
+
+  checkInputs = [
+    click
+    coverage
+    coveralls
+    flake8
+    html5lib
+    jinja2
+    pytest
+    tox
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.cfg --replace "click >=7.0, <8.0" "click >=7.0"
+  '';
```

Comment:
```suggestion
```

Reply:
? https://github.com/NixOS/nixpkgs/blob/e2a1802777ecddcd3fce0d3d066cf8fc8c481dc9/pkgs/development/python-modules/click/default.nix#L19

```
  pname = "click";
  version = "8.1.2";
```

Reply:
hm nevermind, seems it builds successfully anyway